src/lesson7/javaBeginnerChapter2/task26/LargeAppliance.java

```diff
@@ -0,0 +1,7 @@
+package lesson7.javaBeginnerChapter2.task26;
+
+abstract class LargeAppliance extends HomeAppliance {
```

Comment:
You should extend functionality of superclass. But you just create another class similar to superclass:
state and behavior of LargeAppliance  is similar to HomeAppliance

Reply:
fixed